packages/node_modules/@node-red/editor-client/src/js/ui/view.js

```diff
@@ -1957,8 +1961,13 @@ if (DEBUG_EVENTS) { console.warn("clearSelection", mouse_mode); }
                         if (node.g) {
                             var group = RED.nodes.group(node.g);
                             if (!group.selected || group.active) {
-                                RED.group.removeFromGroup(group,node);
-                                node.g = group.id;
+                                var isSelectedGroup = selectedGroups.find(function (ng) {
```

Comment:
Do you still require IE11 support? If so, you cannot use `Array.find` here. It should be enough just to use `indexOf` because the object `group` points at will be the same object held in the `selectedGroups`.

I had to spend some time stepping through this code to work out why it fixed the issue.

I see now the check for `!group.selected` was failing because the `selected` property might have already been reset to `false` on line 1952.

Rather than change the code to do two full passes of `moving_set`, the `if` statement on 1963 could be changed to handle both cases - where `group.selected` is true, or the group is already in `selectedGroups` because it has been handled already.

```
if (!group.selected || group.active || selectedGroups.indexOf(group) > -1) {
```

Reply:
Thanks for pointing that out. Unfortunately, we still need IE11 support.  But, there seems to be other problem with IE11, so I'll check it later.

Regarding the branch condition, I have made some modifications.